hs_access_control/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+from privilege import PrivilegeCodes,\
```

Comment:
This file should be empty right?  None of these imports are actually being used.

Reply:
Since there is no models.py file, these imports are needed here. Ref: https://docs.djangoproject.com/en/1.11/topics/db/models/#organizing-models-in-a-package

Reply:
Yes, that pattern is the intent. It is also followed in hs_core/hydroshare/__init__.py